src/Services/MediaService.php

```diff
@@ -54,7 +55,9 @@ public function create($content, string $fileName, array $data = [], PreviewDriv
             $data['owner_id'] = Auth::check() ? Auth::id() : null;
         }
 
-        $this->createPreviews($fileName, $data, $data['owner_id'], ...$previewDrivers);
+        if (in_array(File::extension(Storage::path($fileName)), config('media.permitted_types.images'))) {

```

Comment:
1. File::extension method just extract the extension of the filename, so no need to pass full Storage::path there
2. It is not the best way to determine if app should generate image preview as we can upload file with any extension  (e.g. `.media`) but with the image mime and package should generate file preview in this case